docs/_static/js/version-alert.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+// Source:
+// https://github.com/anymail/django-anymail/blob/4c443f5515d1d5269a95cb54cf75057c56a3b150/docs/_static/version-alert.js
+
+function warnOnLatestVersion() {
+
+  // The warning text and link is really specific to RTD hosting,
+  // so we can just check their global to determine version:
+  if (!window.READTHEDOCS_DATA || window.READTHEDOCS_DATA.version !== "latest") {
+    return;  // not latest, or not on RTD
+  }
+
+  var warning = document.createElement('div');
+  warning.setAttribute('class', 'admonition danger');
+  warning.innerHTML = "<p class='first admonition-title'>Note</p> " +
+    "<p class='last'> " +
+    "This is <strong>unreleased documentation for Pyodide next version</strong>. " +
+    "For up-to-date documentation, see <a href='/en/stable/'>the stable version</a>." +
```

Comment:
Maybe:
```suggestion
    "<strong>This is documentation for the development version of Pyodide.</strong> " +
    "<a href='/en/stable/'>Click here</a> for documentation about the latest stable version of Pyodide."
```
Might also be nice to provide an instruction for what people should do if they want to try out development APIs. Could be worth looking for another project with a warning like this.

Reply:
Yeah, it could help people who want to try unreleased APIs.
Other projects that I referenced is: [docusaurus](https://docusaurus.io/docs/next), [playwright](https://playwright.dev/docs/next/intro),  [marshmallow](https://marshmallow.readthedocs.io/en/latest/). They don't have such instructions, so I think we should create our own one.